.dockerignore

```
@@ -1,3 +1,4 @@
 build/
 *.egg-info/
 venv/
+.git
```

Let's do it this way since the intended behavior is to block the `.git/` folder and not files and folders that start with `.git`.

```suggestion
.git/
```